### PR TITLE
Custom prometheus integration

### PR DESCRIPTION
**Pull request by NiniiGit**
## Overview
As of now, current documentation only shows how to integrate kubescape with prometheus operator aka kube-prometheus-stack. Sometimes people do not use operator but the actual prometheus community helm chart and customise it as per their need. So allow monitoring of kubescape with such custom/non-operator/community helm chart, we need to make use of  [scraping-pod-metrics-via-annotations](https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus#scraping-pod-metrics-via-annotations) method. So have amended the kubescape helm chart to allow setting up prometheus pod annotation. 

## How to Test
To test the same, once can clone the  PR -branch locally and go to path: `charts/kubescape-prometheus-integrator` and run below command to allow prometheus annotations added. Please check the whole instructions from the updated README file of this PR. After this you would be able to see kubescape in target section of prometheus (installed via community helm chart)

```
helm upgrade --install kubescape-prometheus . -n kubescape-prometheus --set kubescape.prometheusAnnotation.enabled=True --create-namespace
```

## Examples/Screenshots
You would be able to see below prometheus annotations added for kubescape pod deployed via helm chart
```
➜ kubectl describe po kubescape-5d9f6487bd-jm29c |grep -i -A 4 Annotations
Annotations:  prometheus.io/path: /v1/metrics
              prometheus.io/port: 8080
              prometheus.io/scrape: true
Status:       Running

```

### CODE REVIEW

**Review by dwertent — CHANGES_REQUESTED** [review]
Your branch is behind the current `dev` branch. Please fix this :)

Otherwise, well done!

File: charts/kubescape-cloud-operator/Chart.yaml [hunk 1]
```diff
@@ -8,13 +8,15 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.9.2
+
+version: 1.9.5
```
[the remark this answers is not in the document]
Comment (NiniiGit) [reply charts/kubescape-cloud-operator/Chart.yaml]: done, pulled the latest dev branch in this one

**Comment by NiniiGit** [comment]
> Your branch is behind the current `dev` branch. Please fix this :)
> 
> Otherwise, well done!

Hi @dwertent , have added latest dev branch changes. Please could you review once

**Comment by NiniiGit** [comment]
Closing this pull request since have opened a new PR for the same : https://github.com/kubescape/helm-charts/pull/68

**Pull request closed without merging**